src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -133,13 +142,21 @@ export const VetTecContactInformation = ({ institution }) => {
     secondarySCOs.length > 0 && (
       <div className="vads-l-row vads-u-margin-y--4">
         <div className="vads-l-col--12 medium-screen:vads-l-col--3">
-          <h4 className="contact-heading">Secondary</h4>
+          <h4
+            className="contact-heading"
+            aria-label="secondary contact officials"
+          >
+            Secondary
+          </h4>
         </div>
         <div className="vads-l-col--9">
           <div className="vads-l-grid-container--full">
-            <div className="vads-l-row">
+            <ul
+              className="vads-l-row vettec-sco-list vads-u-margin--0"
+              aria-label="secondary contact officials"
```

Comment:
Please change this to an `aria-labelledby` and reference the ID bing added to the `<h4>` above

Reply:
I would first have to get approval by my UX team to change the `<h4>` tags to your recommendation. That's why I used an `aria-label` for the `<ul>` instead of referencing the `<h4>`.